accepted/PSR-7-http-message.md

```diff
@@ -1727,13 +1736,7 @@ interface UriInterface
      *
      * - If a scheme is present, it MUST be suffixed by ":".
      * - If an authority is present, it MUST be prefixed by "//".
-     * - The path can be concatenated without delimiters. But there are two
-     *   cases where the path has to be adjusted to make the URI reference
-     *   valid as PHP does not allow to throw an exception in __toString():
-     *     - If the path is rootless and an authority is present, the path MUST
-     *       be prefixed by "/".
-     *     - If the path is starting with more than one "/" and no authority is
-     *       present, the starting slashes MUST be reduced to one.
+     * - The path can be concatenated without delimiters.
```

Comment:
You can not change the wording of the docblock in an accepted PSR this is against the bylaws. this text needs to be reverted **BUT WE NEED TO EMPHASIZE** in the errata that this is useless with the apply errata.